service_manifest.yml

```diff
@@ -17,21 +17,23 @@ is_external: false
 licence_count: 0
```

Comment:
we should change the accepts to image/.*, assuming we aren't dealing with audiovisual files anymore